doc/source/user_guide/numpy_images.rst

```diff
@@ -147,26 +152,35 @@ denote standard Cartesian coordinates, where ``x`` is the horizontal coordinate,
 ``y`` - the vertical one, and the origin is at the bottom left
 (Matplotlib axes, for example, use this convention).
 
-In the case of multichannel images, the last dimension is used for color channels
-and is denoted by ``channel`` or ``ch``.
+In the case of multichannel images, any dimension (array axis) can be used for
+color channels, and is denoted by ``channel`` or ``ch``. Prior to scikit-image
+0.19, this channels dimension was always last, but in the current release the
+channel dimension can be specified by a ``channel_axis`` argument. Functions
+that require multichannel data default to ``channel_axis = -1``. Otherwise,
+functions default to ``channel_axis = None``, indicating that no axis is
```

Comment:
```suggestion
that require multichannel data default to ``channel_axis=-1``. Otherwise,
functions default to ``channel_axis=None``, indicating that no axis is
```
Just noticed this different convention wrt `doc/source/user_guide/data_types.rst` (showing ``preserve_range=False``). It's no big deal, but I guess the no-space version reflects better the practical use as a function argument?